src/oce_ale_tracer.F90

```diff
@@ -1472,14 +1453,32 @@ FUNCTION bc_surface(n, id, sval, nzmin, partit)
    use recom_glovar
 #endif
   use mod_transit
+  use g_clock
   implicit none
 
   integer,       intent(in)            :: n, id, nzmin
-  real(kind=WP), intent(in)            :: sval
+  real(kind=WP), intent(in)            :: sval, sst, sss, aice
   type(t_partit),intent(inout), target :: partit
+  type(t_mesh),  intent(in), target    :: mesh
+
   REAL(kind=WP)                        :: bc_surface
   character(len=10)                    :: id_string
-  real(kind=WP), dimension(:), pointer :: a_ice
+!  real(kind=WP), dimension(:), pointer :: a_ice  !! MB: where is this needed?
```

Comment:
Should this be commented out? Could be for a coupled setup you dont test. IMO not the right place to remove this line, even if it is not needed.